open_city_profile/graphene.py

```diff
@@ -178,3 +180,26 @@ def __init_subclass_with_meta__(
             _meta=_meta,
             **options,
         )
+
+
+class AllowedDataFieldsMiddleware:
+    def resolve(self, next, root, info, **args):
+        try:
+            is_my_profile = (
+                info.operation.selection_set.selections[0].name.value == "myProfile"
```

Comment:
I wonder if it's sufficient to only filter myProfile queries. A service (e.g. an admin UI of a service) might be using a different query (e.g. `profiles` and `profile`).

Reply:
That's true, I added the those `profile` and `profiles` to be checked as well.